mirroroperator/registrymirror.py

```diff
@@ -134,7 +134,7 @@ def generate_daemon_set(self, daemon_set):
                                     client.V1EnvVar(name="HOSTS_FILE_BACKUP",
                                                     value="/etc/hosts.backup/hosts")
                                 ],
-                                image=self.mirror_hostess_image,
+                                image="{}/ocadotechnology/mirror-hostess:latest".format(self.docker_registry),
```

Comment:
2 things in this line

`latest` should never be used
we need to allow changing the image name completely

So i'd suggest 3 variables: 
- HOSTESS_DOCKER_REGISTRY (docker.io)
- HOSTESS_DOCKER_IMAGE (ocadotechnology/mirror-hostess)
- HOSTESS_DOCKER_TAG (someversion)

then `image="{}/{}:{}".format(self.hostess_docker_registry, self.hostess_docker_image, self.hostess_docker_tag)`

Reply:
Why do we need to allow changing the image name completely?

Reply:
* so that we can build a test image of hostess in our own namespace and try it out
* so that other contributors can build their own version of hostess and use it

possibly a case of YAGNI, but should be a simple change and a change needed to be made anyway